### ISSUE TITLE
types of `useGLTF` missing in `3.9.0`

### ISSUE THREAD

**Issue by NamesMT** [Tresjs/cientos#418]
### Describe the bug

`3.8.0`:
![image](https://github.com/Tresjs/cientos/assets/23612546/2efb552d-3b25-4c3a-92a0-ed1e21883a0d)

`3.9.0`:
![image](https://github.com/Tresjs/cientos/assets/23612546/458d95e5-740f-4139-b5ca-f89e46d5bc45)

It's still exported in .js file and is usable, but the types missing makes IDE yelling error.
![image](https://github.com/Tresjs/cientos/assets/23612546/2a2816d6-7e35-492f-8d8a-f513c725214c)

### Reproduction

https://www.npmjs.com/package/@tresjs/cientos?activeTab=code

### Steps to reproduce

N/A

### System Info

```shell
N/A
```

### Used Package Manager

pnpm

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/Tresjs/cientos/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/Tresjs/cientos/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://cientos.tresjs.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/Tresjs/cientos/issues) that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by wcheek** [comment]
I'm using a temporary hack to get around this issue and still be able to build my project:

```ts
// tresjs-hack.d.ts

// HACK: THIS IS A HACK! REMOVE AFTER TRESJS FIXES TYPES
declare module "@tresjs/cientos" {
  export function useGLTF(any, any);
  export function CameraControls(any);
// Necessary to declare any other cientos packages you are importing here
}
```

**Comment by NamesMT** [comment]
Currently I just simlpy remove opt out of TS (remove `lang="ts"`) on model components :D,
Should be a quick fix but the team is preparing for 4.0 I guess.

**Comment by BenoitBotton** [comment]
> I'm using a temporary hack to get around this issue and still be able to build my project:
> 
> ```ts
> // tresjs-hack.d.ts
> 
> // HACK: THIS IS A HACK! REMOVE AFTER TRESJS FIXES TYPES
> declare module "@tresjs/cientos" {
>   export function useGLTF(any, any);
>   export function CameraControls(any);
> // Necessary to declare any other cientos packages you are importing here
> }
> ```

that worked brilliantly for me

but I am also getting a number of other type errors
error TS2322: Type 'number[]' is not assignable to type 'Vector3 | Readonly<Vector3 | undefined>'
error TS2322: Type 'string' is not assignable to type 'Color | Readonly<Color | undefined>'
which are not resolved.

did anyone else get those?

**Comment by wcheek** [comment]
@BenoitBotton I have to use the classes from `THREE` in those places. 

```ts
:position="new THREE.Vector3(0, 0, 0)"
:color="new THREE.Color('orange')"
```

**Comment by BenoitBotton** [comment]
thanks @NamesMT and @wcheek 
I went with using the THREE classes

**Comment by BenoitBotton** [comment]
actually that does not fully work. one of my components is called several times in a v-for loop. it's template is 
```
<template>
    <TresGroup :rotation-z="cutter.theta * toRad" :key="cutter.spiral">
        <TresGroup :position-z="cutter.ztip">
            <TresGroup :position-x="cutter.radius">
                <TresGroup :rotation-y="cutter.yrot * toRad">
                    <TresGroup :rotation-z="-cutter.siderake * toRad">
                        <TresGroup :rotation-x="cutter.backrake * toRad">
                            <TresMesh :position="cutter.spiral < 500 ? [0, 0, -r] : [0, 0, 0]"
                                :rotation="[90 * toRad, 0, 0]">
                                <Suspense fallback={null}>
                                    <Comp :color="color" />
                                </Suspense>
                            </TresMesh>
                            <TresAxesHelper :visible="showTipHelper" :args="[10]" />
                        </TresGroup>
                    </TresGroup>
                </TresGroup>
            </TresGroup>
        </TresGroup>
    </TresGroup>
</template>
```
only ` :position="cutter.spiral < 500 ? [0, 0, -r] : [0, 0, 0]"` is called out by the type error, not any of the other position or rotation props on the groups.  issue is r is a reactive prop and, when I replace it with
`<TresMesh :position="cutter.spiral < 500 ? new THREE.Vector3(0, 0, -r) : new THREE.Vector3(0, 0, 0)"`
 I end up with only one component showing (or maybe all of them on top of each other? tresjs scene is blank in the dev tools! )

and if I revert back but remove lang=ts from the component, I get nothing at all!

I guess I will revert to cientos 3.8 for now

**Comment by alvarosabu** [comment]
Should be fixed by 4.0 https://github.com/Tresjs/cientos/releases/tag/4.0.0 . If not feel free to reopen
